src/Main.elm

```diff
@@ -143,18 +143,43 @@ loginScreen =
         ]
 
 
-mainScreen : User.User -> H.Html Msg
-mainScreen user =
-    H.div []
-        [ mainScreenHeader user
-        , H.main_ []
-            [ H.div [ HA.class "mdc-layout-grid" ]
-                [ H.div [ HA.class "mdc-layout-grid__inner" ]
-                    [ H.div [ HA.class "mdc-layout-grid__cell" ] []
+mainScreen : AuthedPageState -> H.Html Msg
+mainScreen { user, threads, messageIds } =
+    let
+        allData =
+            RemoteData.map2 (,) threads messageIds
+                |> RemoteData.map
+                    (\( threads, messageIds ) -> ( threads, messageIds ))
```

Comment:
No need to do this `RemoteData.map` because after doing `RemoteData.map2 (,) threads messageIds` you already have tuple inside.

Basically `(,) a b == (a, b)` or in other words `(,)` is a function which gets 2 parameters and return tuple. In the same manner you can use `+` for example: `(+) a b == a + b`.

Though it's probably not a really valuable piece of information since in the next version of the language there will be no `(,)`, `(,,)`, etc functions. So there will be no confusion around them.